python/src/fast.cpp

```diff
@@ -124,7 +124,16 @@ void init_fast(nb::module_& parent_module) {
 
   m.def(
       "scaled_dot_product_attention",
-      &mx::fast::scaled_dot_product_attention,
+      [](const mx::array& queries,
+         const mx::array& keys,
+         const mx::array& values,
+         const float scale,
+         const std::variant<std::monostate, std::string, mx::array>&
+             mask /* = {} */,
+         mx::StreamOrDevice s /*  = {} */) {
```

Comment:
tinies of nits but these parameters don't have default values (it's just a lambda)